### PR TITLE
update chart version

### PR DESCRIPTION
**Pull request by AnilDhawaleGitHub**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -129,6 +129,11 @@ k8s/**/common/ccpay/identity.yaml @hmcts/Fees
 k8s/aat/common/ccpay/ccpay-bulkscanning-app.yaml @hmcts/Fees
 k8s/aat/common/ccpay/ccpay-payment-app.yaml @hmcts/Fees
 
+## bar
+k8s/**/common/bar/bar-app @hmcts/Fees
```
Comment (adusumillipraveen) [inline CODEOWNERS]:
```suggestion
k8s/**/common/bar/bar-app.yaml @hmcts/Fees
```

File: k8s/prod/common/bar/bar-app.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: bar-app
+  namespace: bar
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: bar-api
```
Comment (timja) [inline k8s/prod/common/bar/bar-app.yaml]: shouldn't this be bar-app?

File: k8s/prod/common/bar/bar-app.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: bar-app
```
Comment (timja) [inline k8s/prod/common/bar/bar-app.yaml]: shouldn't this be bar-api?

**Comment by timja** [comment]
conflicts

File: k8s/prod/common/bar/bar-app.yaml (hunk 3 above)
Comment (AnilDhawaleGitHub) [reply k8s/prod/common/bar/bar-app.yaml]: updated

File: k8s/prod/common/bar/bar-app.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    name: bar-api
+    version: 0.2.1
+  values:
+    rollback:
+      enable: true
+    java:
+      replicas: 2
+      image: hmctspublic.azurecr.io/bar/api:prod-4ec01ed4
+      applicationInsightsInstrumentKey: "52125c3c-ea7b-4954-a872-fe6987780a88"
+      keyVaults:
```
[the remark this answers is not in the document]
Comment (AnilDhawaleGitHub) [reply k8s/prod/common/bar/bar-app.yaml]: its done

**Pull request merged**